src/pages/home/ReportScreen.js

```diff
@@ -106,6 +111,9 @@ class ReportScreen extends React.Component {
     }
 
     componentDidUpdate(prevProps) {
+        if (this.state.isSkeletonViewVisible && !this.props.isLoadingReportData) {
+            this.hideSkeletonView();
+        }
```

Comment:
Why do we need both `isLoading` and `isSkeletonViewVisible` and what does it have to do with `isLoadingReportData`?

Reply:
I think it'd be ideal if we could get rid of `isSkeletonViewVisible` and just use `isLoading`. I'm in the same boat as Marc here.

Reply:
I do agree, and I wasn't sure of `isLoading` usage. I've asked a detailed question here https://github.com/Expensify/App/pull/8042#discussion_r863226063. @Luke9389 if you or @marcaaron can chime in then we can take it for closure.

Reply:
Ok let's keep the conversation contained to one thread.